### PR TITLE
#8 BOOKS class file added .

### PR DESCRIPTION
**Pull request by anjali-rathod**
Functionality under book class is present in the code.

### CODE REVIEW

File: books.php [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+   class book
+   {
+      public $id;
+      public $title;
+      public $author;
+      public $category;
+      public $myDate;
+      public $qty;
+      public $available;
+      public $pdo;
```
Comment (guptahemant) [inline books.php:12]: Why not set them protected or private? and add a get method to fetch this data.

File: books.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      public function __construct()
+      {
+         $this->pdo= new PDO('mysql:host=localhost;port=3307;dbname=bookhive', 'userid', 'password');
```
Comment (guptahemant) [inline books.php:15]: use the creds file to get the credentials.

**Review by guptahemant — CHANGES_REQUESTED** [review]
@anjali-rathod couple of small fixes, Apart from that make sure to squash all your commits and add the issue number in front of it.

File: books.php (hunk 2 above)
Comment (AJV009) [reply books.php:15]: And if possible can I have a function which does kind of -> https://github.com/JayKandari/bookhive/issues/8#issuecomment-696330392

**Comment by anjali-rathod** [comment]
Redundant file

**Pull request closed without merging**